### PR TITLE
Add ifyoulovecricket.md

### PR DESCRIPTION
**Pull request by rrhonika**
Please review this, @casey-collab

### CODE REVIEW

File: english/cricket/if-you-love-cricket/ifyoulovecricket.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Love cricket? [Go Here!!](../cricket.md).
```
Comment (casey-collab) [inline english/cricket/if-you-love-cricket/ifyoulovecricket.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by rrhonika** [comment]
Hi! @casey-collab
I have added an option with a link from the original page so my page no reachable.

**Comment by casey-collab** [comment]
Looks great! :crown:

**Pull request merged**